prog/dftb+/lib_dftbplus/mainio.F90

```diff
@@ -2786,7 +2787,8 @@ subroutine writeDetailedOut1(fd, iDistribFn, nGeoSteps, iGeoStep, tMD, tDerivs,
       end if
       if (any(electronicSolver%iSolver == [electronicSolverTypes%qr,&
           & electronicSolverTypes%divideandconquer, electronicSolverTypes%relativelyrobust,&
-          & electronicSolverTypes%elpa, electronicSolverTypes%elpadm])) then
+          & electronicSolverTypes%elpa, electronicSolverTypes%elpadm,&
+          & electronicSolverTypes%pexsi, electronicSolverTypes%ntpoly])) then
```

Comment:
What about introducing an `electronicSolver%providesFreeEnergy` attribute similar to '%providesEigenvalues`?